lib/routes/api/service-providers.js

```diff
@@ -128,6 +128,12 @@ router.get('/:sid/VoipCarriers', async(req, res) => {
     await validateRetrieve(req);
     const service_provider_sid = parseServiceProviderSid(req);
     const results = await VoipCarrier.retrieveAllForSP(service_provider_sid);
```

Comment:
since we are now manipulating this array can you rename it from `results` to `carriers` to make it more clear